apps/web/src/utils/file-upload.ts

```diff
@@ -11,11 +11,14 @@ export function createFileUploadHandler(onSuccess: (file: File) => void) {
     onSuccess,
   }));
 
-  return (event: { target: HTMLInputElement }) => {
-    const [file] = event.target.files ?? [];
+  return [
```

Comment:
There's an issue if we upload a file, remove it and try to upload the same file again. The input's value does not change, so its `onChange` callback is not called, which makes sense.

As a workaround, we can keep a ref to the input element, and in the `onSettled` callback, set its value to an empty string.